src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -22,11 +22,23 @@
   </priorNameHint>
   <priorNameHint>
     <oldName>DSCore.File.ExportToCSV</oldName>
-    <newName>DSCore.IO.File.ExportToCSV</newName>
+    <newName>DSOffice.Data.ExportCSV</newName>
+    <additionalAttributes>
+      <attribute>
+        <name>assembly</name>
+        <value>DSOffice.dll</value>
```

Comment:
This is necessary, otherwise the node will be unresolved (since it's not in `DSCoreNodes.dll`)